src/main/java/mate/academy/service/MovieService.java

```diff
@@ -9,4 +9,8 @@
     Movie get(Long id);
 
     List<Movie> getAll();
+
+    Boolean update(Movie movie);
+
+    Boolean delete(Long id);
```

Comment:
The return type of the `delete` method should be `boolean` instead of `Boolean` to avoid unnecessary boxing.